### PR TITLE
Fix bug : Use try-with-resources or close this "ResultSet" in a "finally" clause

### PR DESCRIPTION
**Pull request by Jave-Chen**
## What is the purpose of the pull request
#1688 
Fix bug : Use try-with-resources or close this "ResultSet" in a "finally" clause

## Brief change log
1.  Reform import sequence (automatically  by  IDE);
2.  Use try-with-resources with ResultSet ;
3.  Fix some code smell .

### CODE REVIEW

**Comment by Jave-Chen** [comment]
QG Failed :
Failed condition 0.0% 0.0% Coverage on New Code (is less than 80%)
Failed condition 9.3% 9.3% Duplicated Lines on New Code (is greater than 3%)

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java [hunk 1]
```diff
@@ -50,26 +50,30 @@ public MonitorRecord getMonitorRecord(Connection conn) {
         Statement pstmt= null;
         try{
             pstmt = conn.createStatement();
-            ResultSet rs1 = pstmt.executeQuery("select count(*) from pg_stat_activity;");
-            while(rs1.next()){
-                monitorRecord.setThreadsConnections(rs1.getInt("count"));
-                break;
+            
+            try (ResultSet rs1 = pstmt.executeQuery("select count(*) from pg_stat_activity;")) {
+                while(rs1.next()){
```
Comment (Technoboy-) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: while ---->  if

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java [hunk 2]
```diff
@@ -50,26 +50,30 @@ public MonitorRecord getMonitorRecord(Connection conn) {
...
+                    break;
+                }
             }
 
-            ResultSet rs2 = pstmt.executeQuery("show max_connections");
-            while(rs2.next()){
-                monitorRecord.setMaxConnections( rs2.getInt("max_connections"));
-                break;
+            try (ResultSet rs2 = pstmt.executeQuery("show max_connections")) {
+                while(rs2.next()){
```
Comment (Technoboy-) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: while --->  if

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java [hunk 3]
```diff
@@ -50,26 +50,30 @@ public MonitorRecord getMonitorRecord(Connection conn) {
...
+                    break;
+                }
             }
 
-            ResultSet rs3 = pstmt.executeQuery("select count(*) from pg_stat_activity pg where pg.state = 'active';");
-            while(rs3.next()){
-                monitorRecord.setThreadsRunningConnections(rs3.getInt("count"));
-                break;
+            try (ResultSet rs3 = pstmt.executeQuery("select count(*) from pg_stat_activity pg where pg.state = 'active';")) {
+                while(rs3.next()){
```
Comment (Technoboy-) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: while --->  if

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java (hunk 1 above)
Comment (Jave-Chen) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: done

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java (hunk 2 above)
Comment (Jave-Chen) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: done

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java (hunk 3 above)
Comment (Jave-Chen) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/PostgrePerformance.java]: done

**Pull request merged**